src/engine.c

```diff
@@ -34,13 +34,48 @@ char *exp_names[MAX_EXPANSION + 1] =
 	"The Brink of War",
 	"Alien Artifacts",
 	"Xeno Invasion",
+	"Rebel vs Imperium (without Gathering Storm)",
 	NULL
 };
 
+int exp_display_order[MAX_EXPANSION] = {0, 1, 2, 6, 3, 4, 5};
+
 /*
  * Maximum number of players per expansion
  */
-int exp_max_player[MAX_EXPANSION] = {4, 5, 6, 6, 5, 5};
+int exp_max_player[MAX_EXPANSION] = {4, 5, 6, 6, 5, 5, 5};
+
+/*
+ * Check whether the expansion can be played with goals.
+ */ 
+inline int expansion_has_goals(int exp)
```

Comment:
yes, it could be done, it would be much cleaner. But it seems to be that it will be a lot of work. Don't know if I can manage this this week-end.

About prev_in_arc vs succ_in_arc, I think there is a catch: in cards.txt, the expansion number corresponds to the material of a box, without relationship with other content. 0 is base game only, 1 is GS only, without base game, ...
In the rest of the code, an expansion is a consistent set of expansion material, eg TGS is base game + TGS material, XI is base game + XI material, ...

So in my mind, prev_in_arc/succ_in_arc does not apply to the same notion of "expansion", and should not be included in this set of information.